src/guard.rs

```diff
@@ -54,13 +54,18 @@
 //!                reason: ErrorReason::InexactByteCount,
 //!            }));
 //! ```
+//! 
+//! Regardless of the chosen strategy, guarded transmutation functions will
+//! always ensure that no out of bounds access is attempted, usually by
+//! restricting the output to spatially safe portions of the input.
 
 
 use error::{ErrorReason, GuardError};
 use std::mem::size_of;
 
 
-/// The `Guard` type describes types which define boundary checking strategies.
+/// The trait describes types which define boundary checking strategies.
+/// See the [module-level documentation](./index.html) for more details.
```

Comment:
Why not `./`, or, indeed, just `.`?

Reply:
I tried that first, but it doesn't route to index.html when reading the docs locally (`cargo doc --open`). `index.html` works too, but that doesn't seem to be conventional.

Reply:
Ah, yes, forgot about people opening the docs as `file://` (btw this is the place where I shill [http](https://crates.io/crates/https)). As for using `index.html`, why doesn't it "seem conventional"? Linking to `file.html` is much more conventional (IME) than `./file.html`, surely?